doc/manual/release_notes.rst

```diff
@@ -1 +1,10 @@
+ARTIQ Releases 
+==============
+
+ARTIQ follows a rolling release model, with beta, stable, and legacy channels. Different releases are saved as different branches on the M-Labs `ARTIQ repository <https://github.com/m-labs/artiq>`_. The ``master`` branch represents the beta version, where at any time the next stable release of ARTIQ is currently in development. This branch is unstable and does not yet guarantee reliability or consistency, but may also already offer new features and improvements; see the `beta release notes <https://github.com/m-labs/artiq/blob/master/RELEASE_NOTES.rst>`_ for the most up-to-date information. The ``release-[number]`` branches represent stable releases, of which the most recent is considered the current stable version, and the second-most recent the current legacy version. 
+
+To install the current stable version of ARTIQ, consult the *current* `Installing ARTIQ <https://m-labs.hk/artiq/manual/installing.html>`_ page. Instructions given in older versions of the manual may or may not install their corresponding ARTIQs, and may or may not currently be supported (e.g. M-Labs does not host ARTIQ versions older than legacy for Conda, and Conda support will probably eventually be removed entirely). Regardless, all out-of-date versions remain available as complete source code on the repository.
```

Comment:
I specifically said older because it's clear to me the beta manual installs the beta, but as of one hour ago the legacy manual didn't seem to be updated. I'll change accordingly...